### PR TITLE
lvs: add Chinese translation

### PR DESCRIPTION
**Pull request by NealSCarffery**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.zh/linux/lvs.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# lvs
+
+> 显示逻辑卷信息.
+> 另见: `lvm`.
+> 更多信息: <https://man7.org/linux/man-pages/man8/lvs.8.html>.
```
Comment (blueskyson) [inline pages.zh/linux/lvs.md:5]:
```suggestion
> 显示逻辑卷信息。
> 另见：`lvm`.
> 更多信息：<https://man7.org/linux/man-pages/man8/lvs.8.html>.
```

File: pages.zh/linux/lvs.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+- 显示逻辑卷信息:
```
Comment (blueskyson) [inline pages.zh/linux/lvs.md:7]:
```suggestion
- 显示逻辑卷信息：
```

File: pages.zh/linux/lvs.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+`lvs`
+
+- 显示所有逻辑卷:
```
Comment (blueskyson) [inline pages.zh/linux/lvs.md:11]:
```suggestion
- 显示所有逻辑卷：
```

File: pages.zh/linux/lvs.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+`lvs -a`
+
+- 改变默认显示以显示更多细节:
```
Comment (blueskyson) [inline pages.zh/linux/lvs.md:15]:
```suggestion
- 改变默认显示以显示更多细节：
```

File: pages.zh/linux/lvs.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+`lvs -v`
+
+- 只显示特定字段:
+
+`lvs -o {{field_name_1}},{{field_name_2}}`
```
Comment (blueskyson) [inline pages.zh/linux/lvs.md:21]:
```suggestion
- 只显示特定字段：

`lvs -o {{域名 1}},{{域名 2}}`
```

File: pages.zh/linux/lvs.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+- 将字段附加到显示:
+
+`lvs -o +{{field_name}}`
```
Comment (blueskyson) [inline pages.zh/linux/lvs.md:25]:
```suggestion
- 将字段附加到显示：

`lvs -o +{{域名}}`
```

File: pages.zh/linux/lvs.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+- 抑制标题行:
```
Comment (blueskyson) [inline pages.zh/linux/lvs.md:27]:
```suggestion
- 抑制标题行：
```

File: pages.zh/linux/lvs.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+`lvs --noheadings`
+
+- 使用特殊分隔符分隔特定字段:
```
Comment (blueskyson) [inline pages.zh/linux/lvs.md:31]:
```suggestion
- 使用特殊分隔符分隔特定字段：
```

**Review by blueskyson — COMMENTED** [review]
Thank you for contribution. Please use full-width punctuations in Chinese sentence and use half-width punctuations to end a sentence if the last word is English. You can refer to [style-guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.zh.md).

File: pages.zh/linux/lvs.md (hunk 2 above)
Comment (navarroaxel) [reply pages.zh/linux/lvs.md:7]: @NealSCarffery Looks like you applied the changes, but you reverted them with the commit `567ba28caaff755114add495480872247fec725a`.

You can apply suggestions from this view without the need to make the commits manually.
Take a look here: <https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes>

If you need help ask us!

Comment (navarroaxel) [reply pages.zh/linux/lvs.md:7]:
```suggestion
- 显示逻辑卷信息：
```

File: pages.zh/linux/lvs.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+
+- 显示所有逻辑卷：
+
+`lvs -a`
+
+- 改变默认显示以显示更多细节：
+
+`lvs -v`
+
+- 只显示特定字段:
```
Comment (navarroaxel) [inline pages.zh/linux/lvs.md:19]:
```suggestion
- 只显示特定字段：
```

File: pages.zh/linux/lvs.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+
+`lvs -o {{域名 1}},{{域名 2}}`
+
+- 将字段附加到显示:
```
Comment (navarroaxel) [inline pages.zh/linux/lvs.md:23]:
```suggestion
- 将字段附加到显示：
```

File: pages.zh/linux/lvs.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
+# lvs
+
+> 显示逻辑卷信息.
+> 另见：`lvm`.
+> 更多信息：<https://man7.org/linux/man-pages/man8/lvs.8.html>.
+
+- 显示逻辑卷信息:
```
Comment (navarroaxel) [inline pages.zh/linux/lvs.md:7]:
```suggestion
- 显示逻辑卷信息：
```

File: pages.zh/linux/lvs.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
+# lvs
+
+> 显示逻辑卷信息.
```
Comment (blueskyson) [inline pages.zh/linux/lvs.md:3]:
```suggestion
> 显示逻辑卷信息。
```

**Review by blueskyson — APPROVED** [review]

File: pages.zh/linux/lvs.md (hunk 12 above)
Comment (blueskyson) [reply pages.zh/linux/lvs.md:3]: Almost done, Thank you @NealSCarffery .

Comment (navarroaxel) [reply pages.zh/linux/lvs.md:3]: I'll apply this quick fix to get merged the first PR of @NealSCarffery :raised_hands:

**Review by navarroaxel — APPROVED** [review]

**Comment by navarroaxel** [comment]
Thanks @NealSCarffery, try to not send PRs from the `main` branch in your fork and keep it in sync with the `tldr-pages:main`, otherwise you can't branch from your `main` branch to create PRs because the branch is dirty.

**Pull request merged**